docs/swagger.yaml

```diff
@@ -31,16 +31,31 @@ paths:
         required: false
         type: "string"
         description: "Filter by status"
+        enum:
+        - "PENDING"
+        - "DUE"
+        - "CHARGED"
+        - "FAILED"
+        - "CANCELLED"
+        - "PARTIALLY_REFUNDED"
+        - "REFUNDED"
+        - "PROCESSING"
       - name: "Authorization"
         in: "header"
         required: true
         type: "string"
-        description: "Bearer 'auth token'"
+        description: "Bearer  'auth token'"
+        maxLength: 512
+        x-example: "Bearer aGFoYWhhaGFoYWhhaGFoYWhoYWhhYWhhaGEK"
       - name: "Merchant-Serial-Number"
         in: "header"
         required: true
         type: "string"
         description: "The applicable merchant serial number currently being represented"
+        minLength: 4
+        maxLength: 10
+        pattern: "^\\d{4,}$"
```

Comment:
MSN set max/min (5.10)?

Reply:
Leaving the length constraints out of the regexp may generate better error messages?
`^\\d*$`

Reply:
@tcskrovseth Hmm, how/why? 🤔

Reply:
MSN is being removed from header since we can get it from the token/do it already :)